packages/api/tests/endpoints/collections/list-collections.js

```diff
@@ -124,3 +152,40 @@ test.serial('returns list of collections with stats when requested', async (t) =
   t.is(results[0].name, t.context.testCollection.name);
   stub.restore();
 });
+
+// Development only, do not merge
+// Test setup creates collections before EACH test so running these
+// in serial will have different results than running in isolation
```

Comment:
Definitely fixable but I didn't want to take the time to refactor the test as this work is only for research purposes and won't be merged.